packages/common/container-definitions/package.json

```diff
@@ -100,34 +100,6 @@
 		"typescript": "~5.1.6"
 	},
 	"typeValidation": {
-		"broken": {
-			"RemovedEnumDeclaration_ContainerErrorType": {
-				"forwardCompat": false,
-				"backCompat": false
-			},
```

Comment:
The removal of ContainerErrorType is expected, from https://github.com/microsoft/FluidFramework/pull/19067/. I see that PR also added the `IPendingLocalState` entry but didn't actually make changes to it, so it must have been removed before that...?

Reply:
Thanks - because the previous version was so outdated, some of the overrides are for types that have since been removed and aren't in either the current or previous versions. I must have mistakenly re-added them in my RC3 bump PR.